charts/workspacemanager/values.yaml

```diff
@@ -171,9 +169,6 @@ buffer:
 # cloudTraceEnabled -- Whether to enable gcp cloud trace
 cloudTraceEnabled: true
 
-# googleFolderId -- the id of the Google Folder to create projects for workspaces within.
-googleFolderId:
```

Comment:
The corresponding env var (`WORKSPACE_GOOGLE_FOLDERID`) should also be removed from the deployment definition, right?

Reply:
Yes, thank you